valkyrie/spec/valkyrie/generators/valkyrie/model_generator_spec.rb

```diff
@@ -9,6 +9,7 @@
     tempdir = ROOT_PATH.join('tmp', 'model')
     Rails::Generators.invoke('valkyrie:model', ['Helm', 'title', 'member_ids:array'], destination_root: tempdir)
     load "#{tempdir}/app/models/helm.rb"
+    module RSpec::Rails; end
```

Comment:
That is to make this check pass: https://github.com/samvera-labs/valkyrie/blob/master/valkyrie/lib/generators/valkyrie/model_generator.rb#L25, so the model generator generates the test spec.  I'll add a comment.